### PR TITLE
LVRUBY-92: Login with facebook

### PR DESCRIPTION
**Pull request by kmandziy**
## JIRA ticket

_Please provide the URL for the JIRA ticket._

## Code reviewers

- [ ] @okohuch 
- [x] @AlinaKovtun 

## Summary of issue

Ability to login with facebook

## Summary of change

Added gem omniauth-facebook
Added controller for omniauth_callbacks

## Review checklist

- [ ] Rubocop doesn't raise any warnings and offenses
- [ ] New features are covered with new specs
- [ ] All specs pass

### CODE REVIEW

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **97.93%** covered at **4.44** hits/line

File: config/routes.rb [hunk 1]
```diff
@@ -11,10 +11,17 @@
...
+    devise_for :users, skip: :omniauth_callbacks, controllers:
+      {
+        registrations: 'devise_overrides/registrations'
+      }
     resources :users
     root to: 'news#index'
     resources :categories
     resources :events
   end
+  devise_for :users, only: :omniauth_callbacks, controllers:
```
Comment (omaik) [inline config/routes.rb]: why separate statement `devise_for`?

File: spec/models/user_spec.rb [hunk 2]
```diff
@@ -69,4 +69,25 @@
...
+      OmniAuth::AuthHash.new(
+        info: {
+          first_name: 'First name',
+          last_name: 'Last name',
+          email: 'facebook@email.com'
+        }
+      )
+    end
+
+    it 'creates new user with if user does not exist' do
```
Comment (omaik) [inline spec/models/user_spec.rb]: it addition would be nice to check whether attributes are assigned correctly to the user

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **97.93%** covered at **4.48** hits/line

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **97.13%** covered at **4.38** hits/line

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **97.13%** covered at **4.38** hits/line

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **97.13%** covered at **4.38** hits/line

**Pull request merged**